src/ui/public/styles/mixins.less

```diff
@@ -1,15 +1,20 @@
 @import "~ui/styles/variables";
 @import (reference) "~bootstrap/less/mixins";
 
-.flex-parent(@grow: 1, @shrink: 1, @basis: auto) {
+.flex-parent(@grow: 1, @shrink: 1, @basis: auto, @direction: column) {
   flex: @grow @shrink @basis;
   display: flex;
-  flex-direction: column;
+  flex-direction: @direction;
 
   > * {
     flex-shrink: 0;
   }
 }
+.real-flex-parent(@display: flex, @flow: column nowrap) {
```

Comment:
does it make sense to have display a param given the function name? maybe we can name this something like flex-parent-flow?